inference-engine/tests/functional/inference_engine/ngraph_reader/custom_op_tests.cpp

```diff
@@ -0,0 +1,243 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <ie_core.hpp>
+#include <ngraph/ngraph.hpp>
+#include <common_test_utils/test_assertions.hpp>
+
+class CustomAddConst : public ngraph::op::Op {
+public:
+    static constexpr ngraph::NodeTypeInfo type_info{"CustomAddConst", 100600};
+    const ngraph::NodeTypeInfo& get_type_info() const override { return type_info;  }
+    CustomAddConst() = default;
+    CustomAddConst(const ngraph::Output<ngraph::Node>& arg, const ngraph::element::Type element_type,
+        const ngraph::Shape shape, const std::shared_ptr<ngraph::runtime::AlignedBuffer> data):
+        ngraph::op::Op({arg}),
+        m_element_type(element_type),
+        m_shape(shape),
+        m_data(data) {
+            constructor_validate_and_infer_types();
+    }
+    void validate_and_infer_types() override {
+        set_output_type(0, m_element_type, m_shape);
+    }
+    std::shared_ptr<ngraph::Node> clone_with_new_inputs(const ngraph::OutputVector& new_args) const override {
+        return std::make_shared<CustomAddConst>(new_args.at(0), m_element_type, m_shape, m_data);
+    }
+    bool visit_attributes(ngraph::AttributeVisitor& visitor) override {
+        visitor.on_attribute("element_type", m_element_type);
+        visitor.on_attribute("shape", m_shape);
+        if (m_data == nullptr) {
```

Comment:
I will suggest to stick with check style like in line 55 `if (!castedNode)`

Reply:
Corrected.